drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -65,6 +66,79 @@ local device_type_profile_map = {
   [ON_OFF_SWITCH_ID] = "switch-binary",
   [ON_OFF_DIMMER_SWITCH_ID] = "switch-level",
   [ON_OFF_COLOR_DIMMER_SWITCH_ID] = "switch-color-level",
+  [GENERIC_SWITCH_ID] = "button"
```

Comment:
Should this change be in this PR? The `button` profile doesn't exist in this PR or `main`

Reply:
Removed in 7462d040a54b3ca6400798f17f6a279af4cc3c05!